console/console-init/ui/src/utils/Formatter.tsx

```diff
@@ -24,9 +24,9 @@ const ConnectionProtocolFormat: React.FunctionComponent<ProtocolIcon> = ({
   encrypted
 }) => {
   return (
-    <>
+    <span>
```

Comment:
is there any specific reason to add span tag.

Reply:
Yes, we have to replace Fragment with some html tag to wrap the content in a single tag, so that nothing get breaks in mobile version